dataservice/api/common/model.py

```diff
@@ -179,4 +180,7 @@ class Base(IDMixin, TimestampMixin):
     """
     Defines base SQlAlchemy model class
     """
-    pass
+    visible = db.Column(db.Boolean(),
+                        nullable=False,
+                        server_default='true',
```

Comment:
Just curious, are you using `server_default` (vs. default) because you specifically want the boolean to be represented as `true` in PG?